### PR TITLE
Adding custom helm values and ingress for ArgoCD and Kubevela

### PR DESCRIPTION
**Pull request by nrajb**
Adding custom helm values and ingress for ArgoCD and Kubevela

### CODE REVIEW

File: deployment/addons/helm-argo/values.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+configs:
```
Comment (shapirov103) [inline deployment/addons/helm-argo/values.yaml:1]: Did you decide against adding ingress in the values.yaml as we discussed as opposed to the separate ingress object?

File: deployment/addons/nginx-app-ingress/hybrid-java-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
```
Comment (shapirov103) [inline deployment/addons/nginx-app-ingress/hybrid-java-ingress.yaml:1]: is this ingress for the java app? Can we move it under applications/java/deployment along with the rest of k8s manifest for this app?

File: deployment/addons/nginx-app-ingress/vela-ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
```
Comment (shapirov103) [inline deployment/addons/nginx-app-ingress/vela-ingress.yaml:1]: Let's move under deployment/addons/vela-core/templates.

**Review by shapirov103 — COMMENTED** [review]
@nrajb looks good please see a few comments.

File: deployment/addons/helm-argo/values.yaml (hunk 1 above)
Comment (nrajb) [reply deployment/addons/helm-argo/values.yaml]: Apologies for the confusion, have corrected it now.

File: deployment/addons/nginx-app-ingress/hybrid-java-ingress.yaml (hunk 2 above)
Comment (nrajb) [reply deployment/addons/nginx-app-ingress/hybrid-java-ingress.yaml]: Yes, moved!

File: applications/java/deployment/java-ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  rules:
+  - host: '*.amazonaws.com'
+    http:
+      paths:
+      - backend:
+          service:
+            name: java-appmod-service
+            port:
+              number: 8080
+        path: /
```
Comment (shapirov103) [inline applications/java/deployment/java-ingress.yaml:20]: This won't work with Argo ingress that we install later - can we change the path to /java-app?

**Review by shapirov103 — COMMENTED** [review]
@nrajb all looks good, see one comment that I added

File: applications/java/deployment/java-ingress.yaml (hunk 4 above)
Comment (nrajb) [reply applications/java/deployment/java-ingress.yaml]: Updated

**Review by shapirov103 — APPROVED** [review]
Look great

**Pull request merged**